store/sqlstore/channel_store.go

```diff
@@ -1419,7 +1419,7 @@ func (s SqlChannelStore) IsUserInChannelUseCache(userId string, channelId string
 
 	ids, err := s.GetAllChannelMembersForUser(userId, true, false)
 	if err != nil {
-		mlog.Error("SqlChannelStore.IsUserInChannelUseCache: " + err.Error())
+		mlog.Error("SqlChannelStore.IsUserInChannelUseCache:", mlog.Err(err))
```

Comment:
Here I would change this error string into something more _human friendly_ like you did in the other case below.